queue_services/entity-filer/src/entity_filer/filing_processors/filing_components/legal_entity_info.py

```diff
@@ -96,8 +92,10 @@ def update_naics_info(legal_entity: LegalEntity, naics: Dict):
     # TODO update NAICS info
     legal_entity.naics_code = naics.get("naicsCode")
     if legal_entity.naics_code:
-        naics_structure = NaicsService.find_by_code(legal_entity.naics_code)
-        legal_entity.naics_key = naics_structure["naicsKey"]
+        # TODO: Uncomment next 2 lines when find_by_code implemented and delete "pass"
+        # naics_structure = NaicsService.find_by_code(legal_entity.naics_code)
+        # legal_entity.naics_key = naics_structure["naicsKey"]
+        pass
```

Comment:
`NaicsService.find_by_code` is not implemented yet, and only returns `None`. I have commented out these two lines of code for now as they throw linter errors.